### ISSUE TITLE
scrollOverflow: true and $.fn.fullpage.setAllowScrolling(false) don't play together nicely

### ISSUE THREAD

**Issue by hbelen** [alvarotrigo/fullPage.js#1584]
Hey all,

I have the following website setup:

![photo_2015-10-09_13-15-50](https://cloud.githubusercontent.com/assets/6825057/10392499/e9d98886-6e87-11e5-9456-43d9b56f5284.jpg)

Every fullpage section has two slides. On the first slide, the amount of content is small enough to fit inside the dimensions of one screen. On the second slide, the amount of content is way bigger, so scrollOverflow is set to true to make scrolling possible.

I would like to disable setAllowScrolling on each second slide, therefore I used the following javascript in the onSlideLeave callback.

> $.fn.fullpage.setAllowScrolling(nextSlideIndex == 0);

On Chrome and Firefox, this works like a charm. On Safari however, scrolling on a second slide doesn't work. It does work after you manually grab the slimscroll bar and move it a little bit with your mouse pointer. It also works if you change the $.fn.fullpage.setAllowScrolling value in the console of the browser directly. I think something goes wrong while initializing the slimscroll plugin in Safari using this setup.

**Comment by alvarotrigo** [comment]
Yep, they are not suppose to. 
Just create a bigger section with `fp-auto-heigh` on that section.

**Comment by hbelen** [comment]
I am not convinced that we are talking about the same concept. Let me try to explain it in greater detail.

I'd like the second slide to act as a 'more information' page for each first slide. The idea is to add a 'back' button in the upper left corner and the bottom of each second slide to return to the first slide.

> Why would you do that?

It's very confusing if a user scrolls to the bottom of a 'more information' second slide, whereafter he get's a transition to the next 'more information' second slide. That's why I want to make it impossible to switch sections if a user is looking at a second slide. At the bottom of the slide the user sees a clear visual cue that the bottom of the slide is reached, with a button to return to slide 1 of the current section.

> It makes no sense to have two scroll bars from the UX perspective.

My idea does not involve two scroll bars. It only involves one scroll bar on each second 'more information' slide, being the slimscroll one. This is the correct usage of the scrollOverflow option.

The only problem is that prohibiting a user from switching sections when he is on a second slide, using the setAllowScrolling method, seems to break the scrollOverflow function in Safari.

Long story short: I am looking for a way to enable scrolling in one slide of a section, but disable switching between sections from that slide. The obvious way to handle this works in Chrome and Firefox, but breaks the scrollOverflow scrolling functionality in Safari.

**Comment by alvarotrigo** [comment]
> I am not convinced that we are talking about the same concept.

Nop, we are not. My fault. I read it very fast and I though you said `$.fn.fullpage.setAutoscrolling(false)`  instead of `$.fn.fullpage.setAllowScrolling(false)`.

I'll open the issue again. 
In any case, it seems a slimScroll problem again. Have you tried using an scrollable element instead ?

**Comment by alvarotrigo** [comment]
I made a demo here: http://jsfiddle.net/97tbk/888/

**Comment by hbelen** [comment]
The demo works as expected, although I found one improvement. You toggle setAutoscrolling in the afterSlideLoad callback, but it's more consistent if you toggle it in the onSlideLeave callback.

Thank you for your quick response!

**Comment by alvarotrigo** [comment]
> You toggle setAutoscrolling in the afterSlideLoad callback, but it's more consistent if you toggle it in the onSlideLeave callback.

Yep, unless you go from Section 1 to Section2 slide 2 :)

**Comment by dogdoy** [comment]
Hi alvarotrigo,

From this demo http://jsfiddle.net/97tbk/888/

If I hover the mouse pointer on the white arrow (on slide 2 page), I can scroll the page as normal.

Is it a bug?

Thank you very much.

**Comment by alvarotrigo** [comment]
@dogdoy yeah. Note that the `afterSlideLoad` callback won't get fired on section change. 
See the console [here](http://jsfiddle.net/97tbk/2394/).
